src/main.c

```diff
@@ -1105,18 +1107,16 @@ static gboolean info_start(int argc, char **argv)
 		goto out;
 	}
 
-	if (bundle->manifest) {
-		manifest = g_steal_pointer(&bundle->manifest);
-	} else {
-		res = load_manifest_from_bundle(bundle, &manifest, &error);
+	if (!bundle->manifest) {
+		res = load_manifest_from_bundle(bundle, &bundle->manifest, &error);
 		if (!res) {
 			g_printerr("%s\n", error->message);
 			g_clear_error(&error);
 			goto out;
 		}
 	}
 
-	text = formatter(manifest);
+	text = formatter(bundle);
```

Comment:
Add a reason for this change to the commit message.

Reply:
I'll replace this by storing the information that the sigdata CMS was encrypted originally in the RaucManifest, too, we can continue using the manifest only first of all.